Client/Layout/MainLayout.razor

```diff
@@ -2,15 +2,41 @@
 
 <RadzenComponents />
 <div class="page">
-    <div class="sidebar">
-        <NavMenu />
-    </div>
-
     <main>
-        <div class = "app_bar">
-            <AppBar/>
+        <div class="header">
+            <h1 class="title">
+                <img src="Logo.png" alt="logo" class="logo" />
+                Contrasting Horizons</h1>
+            <nav>
+                <ul class="nav_links">
```

Comment:
Why was this removed from a separate component? Shouldn't we have a separate component for the navbar?

Reply:
If navbar would be present in all pages so its better it should be shown in the main page as you mentioned in a comment before. The main idea with making something component is that it could be reused but nav bar is the template for all the pages so I think it doesn't need to be a component.